src/webapp/api.py

```diff
@@ -16,26 +16,35 @@
 import logging
 import json
 import datetime
+import PIL
+from PIL import Image
```

Comment:
Preferably, one if these should be used (preferably the first, since Image + other functions can be used by referencing PIL)